keycloak/tasks/main.yaml

```diff
@@ -0,0 +1,60 @@
+---
+- name: Install prerequisites for Keycloak
+  ansible.builtin.apt:
+    name:
+      - openjdk-17-jdk
+      - unzip
+    state: present
+
+- name: Download Keycloak
+  ansible.builtin.get_url:
+    url: "https://github.com/keycloak/keycloak/releases/download/{{ keycloak_version }}/keycloak-{{ keycloak_version }}.zip"
+    dest: "/tmp/keycloak-{{ keycloak_version }}.zip"
+
+- name: Extract Keycloak
+  ansible.builtin.unarchive:
+    src: "/tmp/keycloak-{{ keycloak_version }}.zip"
+    dest: "/opt/"
+    creates: "/opt/keycloak-{{ keycloak_version }}"
+    remote_src: yes
+
+- name: Create a symbolic link for Keycloak
+  ansible.builtin.file:
+    src: "/opt/keycloak-{{ keycloak_version }}"
+    dest: "/opt/keycloak"
+    state: link
+
+- name: Create keycloak user
+  ansible.builtin.user:
+    name: keycloak
+    system: yes
+    shell: /bin/false
+
+- name: Set ownership of Keycloak directory
+  ansible.builtin.file:
+    path: /opt/keycloak
+    state: directory
+    owner: keycloak
+    group: keycloak
+    recurse: yes
```

Comment:
Does keycloak really have to have write access to all its code, or we could only give write access to some part of the directory ?

Reply:
hhhhmm aucune idée, il a peut-être besoin de l'accès qu'à certains sous-répertoires et fichiers, mais je ne saurais pas dire lesquels

Reply:
Vu ensemble sur Slack, droits d'écriture donnés à l'utilisateur keycloak sur /opt/keycloak/data